### PR TITLE
RANCHER-997 Modify automation pipelines to use cypress-cloud

### PR DESCRIPTION
**Pull request by OHaimanov**
https://issues.folio.org/browse/RANCHER-997

### CODE REVIEW

File: pipelines/folioRancher/folioTestingTools/runCypressTests/Jenkinsfile [hunk 1]
```diff
@@ -2,7 +2,8 @@
 import hudson.util.Secret
 import org.jenkinsci.plugins.workflow.libs.Library
 
-@Library('pipelines-shared-library@RANCHER-741-Jenkins-Enhancements') _
+//TODO Set to 'RANCHER-741-Jenkins-Enhancements' before merge
```
Comment (eldiiar-duishenaliev) [inline pipelines/folioRancher/folioTestingTools/runCypressTests/Jenkinsfile:5]: Should it be pointed to custom branch?

File: vars/folioCypressFlow.groovy [hunk 2]
```diff
@@ -21,174 +21,174 @@ import org.folio.Constants
...
+  int numberOfWorkers = params.numberOfWorkers as int ?: 1
+  String agent = params.agent
+  String browserName = "chrome"
+  String cypressImageVersion = ''
+  List resultPaths = []
+
+  buildName customBuildName
+
+
+  timeout(time: testsTimeout, unit: 'HOURS') {
```
Comment (eldiiar-duishenaliev) [inline vars/folioCypressFlow.groovy:49]: Should it(timeout) be under options {} section?

File: pipelines/folioRancher/folioTestingTools/runCypressTests/Jenkinsfile (hunk 1 above)
Comment (OHaimanov) [reply pipelines/folioRancher/folioTestingTools/runCypressTests/Jenkinsfile:5]: yes as it is changes to RANCHER-741-Jenkins-Enhancements

File: vars/folioCypressFlow.groovy (hunk 2 above)
Comment (OHaimanov) [reply vars/folioCypressFlow.groovy:49]: No as we restrict with timeout exact stage related to tests execution

**Review by eldiiar-duishenaliev — APPROVED** [review]

**Review by vkapylou — APPROVED** [review]

**Pull request merged**